wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -935,7 +928,10 @@ extension ZMUserSession: ZMSyncStateDelegate {
 
         managedObjectContext.performGroupedBlock { [weak self] in
             self?.notifyThirdPartyServices()
+            self?.observeMLSGroupVerificationStatus()
+            self?.observeNewCRLsDistributionPoints()
```

Comment:
I'm not sure that it's good idea to move this to `didFinishQuickSync`, we can potentially discover new CRLs during the quick syncing which would then be lost.

cc @David-Henner

Reply:
What bad things happens if we always observe regardless if `e2eiFeature` is enabled or not?

Reply:
> I'm not sure that it's good idea to move this to `didFinishQuickSync`, we can potentially discover new CRLs during the quick syncing which would then be lost.
> 
> cc @David-Henner

Just to clarify, we can potentially discover new CRL **distribution points** during the quick sync. Then we fetch the CRLs from the new distribution points.

We should start observing the new distribution points as soon as possible. During the quick sync, we may forward mls events to CC and CC may return new distribution points. If we start observing after the quick sync then we'd miss these DPs.

> What bad things happens if we always observe regardless if e2eiFeature is enabled or not?

Nothing, I guess CC would never discover any CRL distribution points

Reply:
I moved back the observing of CRL distribution points.